hw/dv/sv/cip_lib/doc/index.md

```diff
@@ -71,6 +71,17 @@ The following is a list of common features and settings:
 * **ral**: This is the instance to the auto-generated RAL model that is
   extended from `dv_base_reg_block`. In the base class, this is created using
   the RAL_T class parameter which the extended IP env cfg class sets correctly.
+* **tl_intg_alert_name**: This the name of the alert that will be triggered once
+  a TLUL integrity error occurs. If the block isn't used the default alert name
+  `fatal_fault`, the user should override it at `initialize`.
+* **tl_intg_alert_fields**: When a TLUL integrity error triggers some of CSR
+  status changed, the user should update this associative array at `initialize` as
+  follows.
+```systemverilog
+tl_intg_alert_fields[reg_name] = field_name;
```

Comment:
```suggestion
* **tl_intg_alert_fields**: An associative array of CSR fields keyed with the 
  corresponding register name. This is the list of CSR fields that are modified 
  when an alert triggers due to TL integrity violation event. The DV user is 
  required to build this list in the `initialize()` method. 

```systemverilog
tl_intg_alert_fields[reg_name] = ral.foo_csr.bar_field;
```

Question: By `reg_name` do you mean the alert status CSR name itself, or is this the reg block name (unique to each TL device interface)?

Reply:
I meant the csr name itself, but I changed my mind to use the object handle as key now, so that we can check if the field path is correct in compile and supply the value as well. Please take a look at #7677.

I also updated the wording as you suggested, PTAL. Thanks